articles/azure-maps/power-bi-visual-manage-access.md

```diff
@@ -15,12 +15,6 @@ services: azure-maps
 
 Power BI provides the ability for designers and tenant administrators to manage the use of the Azure Maps visual. Below you will find steps each role can take.
 
-## Designer options
-
-In Power BI Desktop, designers can disable the Azure Maps visual on the security tab. Select **File** &gt; **Options and settings** and then select **Options** &gt; **Preview features**. When disabled, Azure Maps will not load by default.  
-
```

Comment:
It should be Options and settings > Options > Security

Reply:
I don't understand this comment, this section has been removed based on directions from the Review_tech_doc_for_GA.docx spec:

![image](https://github.com/MicrosoftDocs/azure-docs/assets/33468793/8f1542a2-a696-47b9-9ef1-94b4252daca5)